bundler/lib/bundler/rubygems_ext.rb

```diff
@@ -359,6 +359,27 @@ def extensions_dir
     end
   end
 
+  require "rubygems/name_tuple"
+
+  class NameTuple
+    def self.new(name, version, platform="ruby")
```

Comment:
should this be done in `initialize` instead of `new`?

Reply:
Certain rubies (we won't name names) segfault when I do that. Same if I redeclare `hash` on this object. This is the least segfaulty way since it doesn't undef or overwrite a method.